### ISSUE TITLE
Can't deploy function to Google Cloud Platform

### ISSUE THREAD

**Issue by devLucian93** [GoogleCloudPlatform/cloud-functions-go#37]
Hello,
Thank you for this wonderful initiative to add Go support to Cloud Functions! Unfortunately, I cannot seem to be able to deploy the example function. It works fine locally, but after uploading the zip file to the new function it just gets stuck during deployment and it eventually gets cut off by the quota limit. 
`status: {
   code:  10    
   message:  "Operation interrupted."    
  }`

Any ideas of what might be going wrong? Thanks again!

**Comment by iangudger** [comment]
I wasn't able to reproduce. It may have been a transient issue with the Cloud Functions deployment process. Can you try again?

**Comment by devLucian93** [comment]
I did. I tried to follow the steps in the README. My OS is Ubuntu 18.04, 64bit, kernel version 4.15.0-22-generic. I ran make on my machine and uploaded the zip file to google cloud functions. It just times out when trying to deploy. Is there anything I should change in the standard configuration? (When running make test, I can see the Hello from Go message locally)

**Comment by iangudger** [comment]
Are any errors printed during make? What changes have you made from stock Ubuntu 18.04?

**Comment by devLucian93** [comment]
No errors printed during make. Something weird happened, I tried it again on native Ubuntu after using vagrant and now it seems to deploy fine. Even though I did not change anything. Comparing the 2 zip files I found just this difference between package.json files: 
For the one in vagrant the line is : "execer": "file:local_modules/execer"
For the one in native Ubuntu:  "execer": "file://local_modules/execer"
On Ubuntu I also have node 8.10.0 vs 6.14.3 and gcc 7.3.0 vs 6.3.0
